packages/web-components/src/components/va-select/va-select.tsx

```diff
@@ -71,6 +79,12 @@ export class VaSelect {
 
   @State() options: Array<Node>;
 
+  connectedCallback() {
+    i18next.on('languageChanged', () => {
```

Comment:
To match the React component we could have a `i18next.off('languageChanged')` in a `disconnectedCallback`.

Reply:
Good call 👍